src/api/goods.ts

```diff
@@ -6,6 +6,6 @@ export function getAll(): Promise<Good[]> {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+// export const get5First = () => {};
```

Comment:
By the condition, you should implement this function like below, and it will give you 5 users:
```suggestion
export const load5First = () => {
  return getAll()
    .then(users => users.sort((a, b) => a.name.localeCompare(b.name))
      .slice(0, 5));
};
```

Reply:
fixed, thanks